Sources/Filters/General/Calculator/index.js

```diff
@@ -216,7 +208,7 @@ export function extend(publicAPI, model, initialValues = {}) {
 
 // ----------------------------------------------------------------------------
 
-export const newInstance = macro.newInstance(extend, 'vtkWarpScalar');
+export const newInstance = macro.newInstance(extend, 'vtkCalculator');
```

Comment:
Don't worry I do the same thing. I also fixed some comments too, but at least the comments don't break anything... ;-)